internal/template/method.go

```diff
@@ -179,12 +179,18 @@ func ({{.S}} {{.NewStructName}}Do) Assign(attrs ...field.AssignExpr) *{{.NewStru
 	return {{.S}}.withDO({{.S}}.DO.Assign(attrs...))
 }
 
-func ({{.S}} {{.NewStructName}}Do) Joins(field field.RelationField) *{{.NewStructName}}Do {
-	return {{.S}}.withDO({{.S}}.DO.Joins(field))
-}
-
-func ({{.S}} {{.NewStructName}}Do) Preload(field field.RelationField) *{{.NewStructName}}Do {
-	return {{.S}}.withDO({{.S}}.DO.Preload(field))
+func ({{.S}} {{.NewStructName}}Do) Joins(field ...field.RelationField) *{{.NewStructName}}Do {
+	for _, f := range field {
```

Comment:
It would be better to name the loop variable with prefix "_" to avoid name conflict with `{{.S}}`
```go
for _, _f := range fields{
```